python/cuml/neighbors/kneighbors_classifier.pyx

```diff
@@ -125,9 +125,8 @@ class KNeighborsClassifier(NearestNeighbors,
 
       knn = KNeighborsClassifier(n_neighbors=10)
 
-      X_train, X_test, y_train, y_test =
-        train_test_split(X, y, train_size=0.80)
-
+      X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=0.80)
```

Comment:
```suggestion
      X_train, X_test, y_train, y_test = \
          train_test_split(X, y, train_size=0.80)
```

this will avoid going over 80 characters for pep8 compliance